cmd/spawnersvc/main.go

```diff
@@ -94,7 +94,7 @@ func startSignalHandler(g *group.Group) {
 
 func main() {
 
-	err := config.Load(".")
+	err := config.Load("./../../")
```

Comment:
dont change this to relative to where main is located, the binary will be executed at the same path as config.

Reply:
updated